app/presenters/summary/sections/client_other_charge.rb

```diff
@@ -0,0 +1,44 @@
+module Summary
+  module Sections
+    class ClientOtherCharge < Sections::BaseSection
+      def show?
+        kase.present? && kase.client_other_charge_in_progress.present? && super
+      end
+
+      def answers # rubocop:disable Metrics/MethodLength,Metrics/AbcSize
+        [
+          Components::ValueAnswer.new(
+            :client_other_charge_in_progress, kase.client_other_charge_in_progress,
+            change_path: edit_steps_case_other_charge_in_progress_path(subject: 'client')
+          ),
+          Components::FreeTextAnswer.new(
+            :client_other_charge_charge, kase.client_other_charge&.charge,
+            change_path: edit_steps_case_other_charge_in_progress_path(subject: 'client'),
```

Comment:
The routing here should be `edit_steps_case_other_charge_path` and the same for the questions below

Reply:
Well spotted!